usage/cli/src/main/java/brooklyn/cli/Main.java

```diff
@@ -266,7 +290,7 @@ private String readLine(InputStream in) throws IOException {
         // looks like: {@linktourl https://gist.github.com/47066f72d6f6f79b953e}
         @Beta
         @Option(name = { "-sk", "--stopOnKeyPress" },
-                description = "After startup, shutdown on user text entry (default false)")
+                description = "Shutdown immediately on user text entry after startup (useful for debugging and demos)")
```

Comment:
Definite improvement; we should also remove mention of Whirr in comment above this (can be in a separate PR)